test-infra/infra-down.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
```

Comment:
Is there a whitespace between #! and /bin/sh? Shouldn't this be `#!/bin/sh` or is this still a valid shebang line? oO

Reply:
removed whitespace. Strange it did not error out.